test.sh

```diff
@@ -1,30 +1,75 @@
 #!/bin/bash
 
-set -evu
```

Comment:
please return this it's essential for CI, `-v` is important to see the output in GitLab, `-u` fails on unset vars and `-e` fails on errors.

Reply:
returned